src/ConnectedMode.UnitTests/UI/ManageBinding/ManageBindingViewModelTests.cs

```diff
@@ -492,6 +496,29 @@ await progressReporterViewModel.Received(1)
                     x.WarningText == UiResources.LoadingConnectionsFailedText &&
                     x.AfterProgressUpdated == testSubject.OnProgressUpdated));
     }
+    
+    [TestMethod]
```

Comment:
This test it monstrous. Please split it into multiple tests for the following cases:

- SolutionInfo is filled for Solution
- SolutionInfo is filled for FolderWorkspace
- BoundServerProject is null
- BoundServerProject is NOT null and the response succeeded
- BoundServerProject is NOT null and the response failed